dm/config/task.go

```diff
@@ -656,39 +658,48 @@ func (c *TaskConfig) FromSubTaskConfigs(stCfgs ...*SubTaskConfig) {
 	c.Loaders = make(map[string]*LoaderConfig)
 	c.Syncers = make(map[string]*SyncerConfig)
 
+	BAListMap := make(map[string]string, len(stCfgs))
```

Comment:
`baList`

Reply:
We use BAList in task config. https://github.com/pingcap/dm/blob/4ebd3b5de1e4e7ba49faaae508d671e0b4df03df/dm/config/task.go#L318

Reply:
IMO that's public accessible so could use uppercase